apis/comms_api/routers/router.py

```diff
@@ -0,0 +1,10 @@
+from fastapi import APIRouter, status
+from fastapi.responses import Response
+
+
+router = APIRouter(prefix='/api/v1')
+
+
+@router.get("/")
+async def home():
+    return Response(status_code=status.HTTP_200_OK)
```

Comment:
For the root endpoint (`/`) the  management api is returning something like:

```json
{
"title": "Wazuh API",
"api_version": "v4.8.1",
"revision": "40100",
"license_name": "GPL 2.0",
"license_url": "https://github.com/wazuh/wazuh/blob/v4.8.1/LICENSE",
"hostname": "wazuh",
"timestamp": "2019-04-02T08:08:11Z"
}
```
Mabe can we implement something similar here.

Reply:
The Server management API is loading those values from the spec:

```py
async def default_info(pretty: bool = False) -> ConnexionResponse:
    ...
    info_data = load_spec()
    data = {
        'title': info_data['info']['title'],
        'api_version': info_data['info']['version'],
        'revision': info_data['info']['x-revision'],
        'license_name': info_data['info']['license']['name'],
        'license_url': info_data['info']['license']['url'],
        'hostname': socket.gethostname(),
        'timestamp': get_utc_now().strftime(DATE_FORMAT)
    }
    data = WazuhResult({'data': BasicInfo.from_dict(data)})

    return json_response(data, pretty=pretty)
```

We could implement something similar for this one by creating a metadata file that is updated automatically, but I'm not sure if that is something we want, or if there is going to be a global source where to get that information source.

In any case, I would prefer to do this later in the development to avoid spending time on a solution that will be discarded later.